src/schema/types/mutations/Project.ts

```diff
@@ -17,10 +26,13 @@ export const updateProject = {
     id: nonNull(intArg()),
     project: arg({ type: "ProjectUpdateInput" }),
   },
-  resolve(_: any, args: any, ctx: any) {
-    // 후에 프로젝트 수정 구현
-    const mock_link = "http://mock-example.com";
-    return mock_link;
+  resolve: async (_: any, args: any, ctx: any) => {
```

Comment:
해당 `resolve`에  `ctx`를 사용 안하니 `_`처리 하는 것이 좋다고 생각하는데, 어떻게 생각하세요?
```suggestion
  resolve: async (_: any, args: any, __: any) => {
```